devel/shiboken6/Makefile

```diff
@@ -0,0 +1,48 @@
+# $FreeBSD$
+
+PORTNAME=	shiboken6
+DISTVERSION=	6.0.0 # ${QT6_VERSION} ?
+CATEGORIES=	devel
+MASTER_SITES=	QT/official_releases/QtForPython/${PORTNAME}/PySide6-${DISTVERSION}-src
+PKGNAMEPREFIX=	${PYTHON_PKGNAMEPREFIX}
+DISTNAME=	pyside-setup-opensource-src-${DISTVERSION}
+
+MAINTAINER=	kde@FreeBSD.org
+COMMENT=	Python binding generator for C++ libraries
+
+LICENSE=	GPLv2 LGPL21 LGPL3                  
+LICENSE_COMB=	multi
+
+BUILD_DEPENDS=	${LOCALBASE}/include/vulkan/vulkan.h:devel/vulkan-headers \
+		llvm${LLVM_DEFAULT}>0:devel/llvm${LLVM_DEFAULT} \
+		git>0:devel/git \
+		${PYTHON_PKGNAMEPREFIX}sphinx>0:textproc/py-sphinx
+LIB_DEPENDS=	libvulkan.so:graphics/vulkan-loader \
+		libLLVM.so:devel/llvm${LLVM_DEFAULT} \
+		libxslt.so:textproc/libxslt
+
+USES=		cmake compiler:c++17-lang gnome localbase python:3.5+ \
+		shebangfix tar:xz qt:6
+USE_GNOME=	libxml2
+USE_LDCONFIG=	yes
+USE_PYTHON=	flavors
+USE_QT=		5compat base declarative doc imageformats quick3d quickcontrols2 quicktimeline networkauth shadertools svg tools translations wayland
+
+# TODO: avoid a conflict with shiboken2
+SHEBANG_FILES=	shiboken_tool.py
+
+CMAKE_ARGS+=	-DClang_DIR:PATH="${LOCALBASE}/llvm${LLVM_DEFAULT}/lib/cmake/clang" \
+		-DLLVM_DIR:PATH="${LOCALBASE}/llvm${LLVM_DEFAULT}/lib/cmake/llvm" \
+		-DCMAKE_CXX_FLAGS=-lexecinfo \
```

Comment:
hm, this should be patched in the CMakeFile and upstreamed, using [1]

[1] https://cmake.org/cmake/help/latest/module/FindBacktrace.html